pages/starships/[id].js

```diff
@@ -82,7 +81,7 @@ export default function Starship() {
           <StarshipInfo label="Starship class:" value={starship_class} />
         </Flex>
       </Flex>
-      <StarshipSellerList img={img} name={name} id={_id} />
+      <StarshipSellersList img={img} name={name} starshipId={_id} />
     </Flex>
```

Comment:
We (we looked at this as a team) don't fully understand the decision/s you've made to put some styling into components and some into the pages like this. We certainly don't (yet?) think of ourselves as professionals, but we think your code would be cleaner if you exported your styling to components and kept your code on each page to an easily-looked-at minimum.

Reply:
Some of the components are very simple and so, we decided to keep some of them on the page. But you are right, some of them should be a extra component. We will invest more time at the end of the project to make the css optimally.